src/ui/public/chrome/directives/kbn_chrome.js

```diff
@@ -44,7 +47,9 @@ export default function (chrome, internals) {
         onRouteChange();
 
         // and some local values
-        chrome.httpActive = $http.pendingRequests;
+        $scope.$watch(() => $http.pendingRequests, (pendingHttpRequests) => {
+          chrome.httpActive = pendingHttpRequests.filter(request => !systemApi.isSystemApiRequest(request));
+        }, true);
```

Comment:
Right. I found that the watch function (2nd argument) wasn't being triggered without the deep watch. Perhaps using `$watchCollection` will solve that issue...